### PR TITLE
[Footer] - add a component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-793

### CODE REVIEW

File: packages/ui/src/2_molecules/Footer/Footer.module.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      @apply my-0;
+    }
+  }
+
+  .leftContent {
+    @apply flex-row;
+  }
+
+  .links {
+    @apply ml-14 mt-0;
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/Footer/Footer.module.css]:
```suggestion
    @apply mt-0;
```

File: packages/ui/src/2_molecules/Footer/Footer.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  dataLayoutId: '',
+  leftContent: (
+    <Link
+      href="/"
+      text={<img className="max-h-4" src={SovrynLogo} alt="Sovryn logo" />}
+    />
+  ),
+  links: (
+    <>
+      <Link href="/zero" text="Zero" className="mr-2" />
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/Footer/Footer.stories.tsx]:
```suggestion
      <Link href="/zero" text="Zero" className="mr-4" />
```

File: packages/ui/src/2_molecules/Footer/Footer.module.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.leftContent,
+.rightContent,
+.links {
+  @apply flex items-center;
+}
+
+.leftContent {
+  @apply flex-col;
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/Footer/Footer.module.css]:
```suggestion
  @apply flex-col gap-x-14;
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: packages/ui/src/2_molecules/Footer/Footer.stories.tsx (hunk 2 above)
Comment (Rickk137) [reply packages/ui/src/2_molecules/Footer/Footer.stories.tsx]: Screen Shot 2022-11-11 at 2 28 40 PM

File: packages/ui/src/2_molecules/Footer/Footer.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      <Link href="/perpetuals" text="Perpetuals" />
+    </>
+  ),
+  rightContent: (
+    <>
+      {socials.map(item => (
+        <LinkBase
+          key={item.id}
+          className="ml-2 border border-white/10 text-sov-white rounded-full w-6 h-6 p-0.5 flex justify-center items-center hover:bg-gray-80"
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/Footer/Footer.stories.tsx]: p-1.5

**Review by Rickk137 — CHANGES_REQUESTED** [review]

**Review by Rickk137 — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Review by LunaSov — APPROVED** [review]

**Pull request merged**